docs/using-environments.rst

```diff
@@ -120,25 +109,20 @@ The method is parameterized with a problem instance file: the combinatorial
 optimization problem that will be loaded and solved by the `SCIP <https://scip.zib.de/>`_ solver
 during the episode.
 
-* The ``observation`` consists in relevant information extracted from the solver,
-  for the user to decide what the next action will be (typically
-  using a machine learning algorithm).
-* The ``action_set``, if present, describes the set of candidate
-  actions which are valid for the next transition. This allows us to deal with highly dynamic
-  actions sets in a simple way.
+* The ``observation`` consists of information about the state of the solver that should be used to select the next action 
+  to perform (for example, using a machine learning algorithm.)
+* The ``action_set``, when non-null, describes the set of candidate actions which are valid for the next transition. 
```

Comment:
non-null might be a bit misleading here. What value does the action_set take when it is not used ? None ?

Reply:
In python, `None`, yes. Maybe we should say that instead